frontend/src/metabase-lib/expressions/resolver.js

```diff
@@ -152,6 +158,11 @@ export function resolve(expression, type = "expression", fn = undefined) {
     if (!clause) {
       throw new ResolverError(t`Unknown function ${op}`, expression.node);
     }
+
+    if (database && !database.hasFeature(clause)) {
```

Comment:
`clause` is an object, `Database.prorotype.hasFeature` accepts a `string`.
Currently it's not possible to use any function :see_no_evil: 

```suggestion
    if (database && !database.hasFeature(clause.displayName)) {
```

Reply:
My brain must've been turned of when I wrote this. I've fixed this and added database to the recursive `resolve` calls too.